File: src/JS/Exercise_065.js

// Area or Perimeter
// 8kyu
// https://www.codewars.com/kata/5ab6538b379d20ad880000ab
// 2022-05-25 T:21:48:13

const areaOrPerimeter = function (l, w) {
    return l === w ? l * w : 2 * l + 2 * w
};

File: src/primus/pawel/Exercise_065.java
package primus.pawel;


import java.util.Collections;

// String repeat
// 8kyu
// https://www.codewars.com/kata/57a0e5c372292dd76d000d7e/train/java
// 2021-04-11 T:20:32:33
class Exercise_065 {

    public static String repeatStr(final int repeat, final String string) {
        return String.join("", Collections.nCopies(repeat, string));
    }
}
